### PR TITLE
consistent fake embeddings

### PR DESCRIPTION
**Pull request by efriis**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
-langchain-core = ">=0.0.12"
+langchain-core = "^0.1.33"
```
Comment (efriis) [inline pyproject.toml:10]: you were on reallly old core + langchain versions. Fixed here.

File: tests/integration_tests/test_vectorstores.py [hunk 2]
```diff
@@ -780,7 +780,7 @@ def test_autocut(weaviate_client, embedding_openai, auto_limit, expected_num_doc
 
     docsearch = WeaviateVectorStore.from_texts(
         texts=texts,
-        embedding=embedding_openai,
+        embedding=ConsistentFakeEmbeddings(),
```
Comment (efriis) [inline tests/integration_tests/test_vectorstores.py:783]: and you can do this in any others that want to call openai embeddings!

File: tests/integration_tests/fake_embeddings.py
Comment (efriis) [inline tests/integration_tests/fake_embeddings.py:1]: maps texts to your desired embedding

**Review by hsm207 — APPROVED** [review]
Thanks for helping!

**Pull request merged**